dev/NavigationView/NavigationView_InteractionTests/PaneBehaviorTests.cs

```diff
@@ -951,5 +951,61 @@ public void EnsureDynamicSizeForPaneHeaderFooterAndCustomContent()
                 Verify.IsTrue(paneCustomContentButton.BoundingRectangle.Width <= widthCompactBoundary && paneCustomContentButton.BoundingRectangle.Width > 0);
             }
         }
+    
+    
+        [TestMethod]
+        public void SelectingAnItemInLeftCompactOrLeftMinimalClosesPane()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView Test" }))
+            {
+                var configs = new ControlWidth[] { ControlWidth.Narrow, ControlWidth.Medium };
+
+                foreach(var width in configs)
+                {
+                    SetNavViewWidth(width);
+
+                    CheckBox isPaneOpenCheckBox = new CheckBox(FindElement.ById("IsPaneOpenCheckBox"));
+
+                    // On phone, the pane will initially be in the closed compact state, so open it before
```

Comment:
>phone [](start = 26, length = 5)

is this copied from somewhere? we don't test on phone anymore.

Reply:
Yes I copied the skeleton of another test, updated the comment.